src/app/games/dvonn/tests/dvonn.component.spec.ts

```diff
@@ -114,21 +114,37 @@ describe('DvonnComponent', () => {
         await testUtils.expectClickFailure('#click_3_3', DvonnFailure.MUST_MOVE_IN_STRAIGHT_LINE());
     }));
     it('should allow selecting another piece when one is already selected (invalid move)', fakeAsync(async() => {
-        // Given that the user has selected a piece
-        await testUtils.expectClickSuccess('#click_0_4');
+        // Given a board where the user has selected a piece
+        const board: Table<DvonnPieceStack> = [
+            [__, __, W_, __, __, __, __, __, __, __, __],
+            [__, __, D_, W_, W_, __, __, __, __, __, __],
+            [__, __, W_, __, __, __, __, __, __, __, __],
+            [__, __, __, __, __, __, __, __, __, __, __],
+            [__, __, __, __, __, __, __, __, __, __, __],
+        ];
+        testUtils.setupState(new DvonnState(board, 0, false));
+        await testUtils.expectClickSuccess('#click_2_0');
         // When the user selects a invalid destination, which is a valid selection for another piece
-        await testUtils.expectClickSuccess('#click_2_4');
+        await testUtils.expectClickSuccess('#click_4_1');
```

Comment:
When the user clicks on an invalid destination for this move that could be a valid starting coord for another move ?